### ISSUE TITLE
Raise errors like TypeError

### ISSUE THREAD

**Issue by xychen1015** [gjzheng93/frap-pub#1]
Hi, after setup, I got the following errors after I ran the program.
![image](https://user-images.githubusercontent.com/52397468/73719121-60ce0f80-4759-11ea-8ffa-06e6d71c4d3a.png)
![image](https://user-images.githubusercontent.com/52397468/73719140-6fb4c200-4759-11ea-9ec4-1df387675f3c.png)
Besides, when installing cityflow, should I modify the package name 'engine' to 'cityflow' in the file anon_env.py

**Comment by aaron851113** [comment]
1. You should install docker :
$ docker pull gjzheng/frap:latest
2. Then run this docker :  
$ docker run -t -i gjzheng/frap
3. cd to file
$ cd home/
4.  Git clone this github 
$ git clone https://github.com/gjzheng93/frap-pub/
$ cd frap-pub
5. run the bash
$ bash example_run.sh

6. Then the docker auto training the cityflow model , not need to modify 'engine' package.

![image](https://user-images.githubusercontent.com/48345009/74902087-ec919e00-53df-11ea-98ae-74b9445fb4e8.png)

**Comment by gjzheng93** [comment]
Yes. Please follow the steps that aaron has proposed. We recommend to use the docker. That is easier.

**Comment by luxueliang1996** [comment]
Hello, why is my reply.txt empty and how can I make it run in cityflow
